cypress/support/hub-commands.ts

```diff
@@ -106,48 +106,51 @@ Cypress.Commands.add('uploadHubCollectionFile', (hubFilePath: string, hubFileNam
+      `/api/galaxy/v3/plugin/ansible/search/collection-versions/?is_deprecated=false&repository_label=!hide_from_search&is_highest=true&keywords=ds8000&offset=0&limit=10`
     ).then((result) => {
-      const collectionC = result.data.length;
+      const collectionB = result.data.length;
       cy.requestGet<HubItemsResponse<CollectionVersionSearch>>(
-        `/api/galaxy/v3/plugin/ansible/search/collection-versions/?is_deprecated=false&repository_label=!hide_from_search&is_highest=true&keywords=ds8000&offset=0&limit=10`
+        `/api/galaxy/v3/plugin/ansible/search/collection-versions/?is_deprecated=false&repository_label=!hide_from_search&is_highest=true&keywords=operator_collection_sdk&offset=0&limit=10`
       ).then((result) => {
-        const collectionD = result.data.length;
-        cy.requestGet<HubItemsResponse<CollectionVersionSearch>>(
-          `/api/galaxy/v3/plugin/ansible/search/collection-versions/?is_deprecated=false&repository_label=!hide_from_search&is_highest=true&keywords=operator_collection_sdk&offset=0&limit=10`
-        ).then((result) => {
-          const collectionE = result.data.length;
-          if (collectionB === 0) {
-            newCollectionVersion = 'ibm-zos_cics_operator-1.0.1.tar.gz';
-            return newCollectionVersion;
-          } else if (collectionC === 0) {
-            newCollectionVersion = 'ibm-spm_toolbox-1.0.2.tar.gz';
-            return newCollectionVersion;
-          } else if (collectionD === 0) {
-            newCollectionVersion = 'ibm-ds8000-1.1.0.tar.gz';
-            return newCollectionVersion;
-          } else if (collectionE === 0) {
-            newCollectionVersion = 'ibm-operator_collection_sdk-1.1.0.tar.gz';
-            return newCollectionVersion;
-          } else {
-            return 'All test collections currently exist. Please delete one or more and re-run the test.';
-          }
-        });
+        const collectionC = result.data.length;
+        if (collectionA === 0) {
+          newCollectionVersion = 'ibm-spm_toolbox-1.0.2.tar.gz';
+          return newCollectionVersion;
+        } else if (collectionB === 0) {
+          newCollectionVersion = 'ibm-ds8000-1.1.0.tar.gz';
+          return newCollectionVersion;
+        } else if (collectionC === 0) {
+          newCollectionVersion = 'ibm-operator_collection_sdk-1.1.0.tar.gz';
+          return newCollectionVersion;
+        } else {
+          return 'All test collections currently exist. Please delete one or more and re-run the test.';
+        }
       });
     });
   });
-  // });
 });
 
+Cypress.Commands.add(
+  'deleteCollectionFromSystem',
+  (
+    collectionName: string,
+    options?: {
+      /** Whether to fail on response codes other than 2xx and 3xx */
+      failOnStatusCode?: boolean;
+    }
+  ) => {
+    cy.requestDelete(
+      `/api/galaxy/v3/plugin/ansible/content/community/collections/index/ibm/${collectionName}/`,
```

Comment:
We need to use hubAPI or pulpAPI here, since the URL could differ based on the deployment type.